app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -26,15 +27,20 @@ export default function NoteTable({
                 }}
               />
             </th>
-            <th className="text-left">Title</th>
-            <th className="text-left">Description</th>
+            <th className="text-left text-gray-400">Title</th>
+            <th className="text-left text-gray-400">Description</th>
+            <th className="text-center text-gray-400">Tags</th>
+            <th className="text-center text-gray-400">Created Date</th>
+            <th className="text-center text-gray-400">Due Date</th>
+            <th className="text-center text-gray-400">Contact</th>
+            <th></th>
```

Comment:
Please remove all overrides and pass `nui-table nui-table--hover nui-table--actions` as `classnames` to the table.

`nui-table--hover` and `nui-table--actions` add the required hover effects to the table row and `IconButtons`.